services/base.py

```diff
@@ -414,4 +478,19 @@ def isCorrectOrientation(self, metadata, displaySize):
 
     return image_orientation == display_orientation
 
+  def nextAlbum(self):
```

Comment:
Comment to explain for newcomers what this does

Reply:
done